components/button/button-move.js

```diff
@@ -0,0 +1,211 @@
+ */
+class ButtonMove extends ButtonMixin(RtlMixin(LitElement)) {
+
+	static get properties() {
+		return {
+			/**
+			 * A description to be added to the button for accessibility when text on button does not provide enough context
+			 * @type {string}
+			 */
+			description: { type: String },
+
+			/**
+			 * REQUIRED: Accessible text for the button
+			 * @type {string}
+			 */
+			text: { type: String, reflect: true }
+
+		};
+	}
+
+	static get styles() {
+		return [ buttonStyles,
+			css`
+				:host {
+					display: inline-block;
+					line-height: 0;
+				}
+				:host([hidden]) {
+					display: none;
+				}
+				button {
+					background-color: transparent;
+					display: flex;
+					flex-direction: column;
+					gap: 2px;
+					margin: 0;
+					min-height: 1.8rem;
+					padding: 0;
+					position: relative;
+					width: 0.9rem;
+				}
+				d2l-icon {
+					border-radius: 0.1rem;
+					height: 0.85rem;
+					width: 0.9rem;
+				}
+				button:focus {
+					background-color: #ffffff;
+				}
+				button:hover > d2l-icon,
+				button:focus > d2l-icon {
+					background-color: var(--d2l-color-mica);
+				}
+				.up-icon {
+					border-top-left-radius: 0.3rem;
+					border-top-right-radius: 0.3rem;
+				}
+				.down-icon {
+					border-bottom-left-radius: 0.3rem;
+					border-bottom-right-radius: 0.3rem;
+				}
+
+				.up-layer,
+				.down-layer {
+					height: 1.1rem;
+					left: -0.2rem;
+					position: absolute;
+					width: 1.3rem;
+				}
+				.up-layer {
+					top: -0.25rem;
+				}
+				.down-layer {
+					bottom: -0.25rem;
+				}
+				:host([dir="rtl"]) .up-layer,
+				:host([dir="rtl"]) .down-layer {
+					left: auto;
+					right: -0.2rem;
+				}
+
+
+				/* Firefox includes a hidden border which messes up button dimensions */
+				button::-moz-focus-inner {
+					border: 0;
+				}
+				:host([disabled]) button {
+					cursor: default;
+					opacity: 0.5;
+				}
+				button[disabled]:hover > d2l-icon {
+					background-color: transparent;
+				}
+			`
+		];
+	}
+
+	constructor() {
+		super();
+
+		/** @internal */
+		this._buttonId = getUniqueId();
+		/** @internal */
+		this._describedById = getUniqueId();
+	}
+
+	render() {
+		return html`
+			<button
+				aria-describedby="${ifDefined(this.description ? this._describedById : undefined)}"
+				aria-disabled="${ifDefined(this.disabled && this.disabledTooltip ? 'true' : undefined)}"
+				aria-label="${this.ariaLabel ? this.ariaLabel : ifDefined(this.text)}"
+				?autofocus="${this.autofocus}"
+				?disabled="${this.disabled && !this.disabledTooltip}"
+				id="${this._buttonId}"
+				@keydown="${this._handleKeydown}"
+				name="${ifDefined(this.name)}"
+				title="${ifDefined(this.text)}"
+				type="${this._getType()}">
+				<d2l-icon icon="tier1:arrow-toggle-up" class="up-icon"></d2l-icon>
+				<d2l-icon icon="tier1:arrow-toggle-down" class="down-icon"></d2l-icon>
+				<div class="up-layer" @click="${this._handleUpClick}"></div>
```

Comment:
As mentioned in previous PR, these `div` layers are rendered on top of the icons and extend into the margin where the focus outline is displayed, so that it meets the minimum WCAG 2.2 - 2.5.8 Target Size criteria.